gradle.properties

```diff
@@ -20,4 +20,6 @@ SAMPLE_VERSION_NAME=0.0.37
 
 KIN_ECOSYSTEM_SDK_VERSION_NAME=0.0.11
 
-KIN_ECOSYSTEM_CORE_VERSION_NAME=0.0.1
+KIN_ECOSYSTEM_CORE_VERSION_NAME=0.0.2
+
+KIN_ECOSYSTEM_COMMON_VERSION_NAME=0.0.1
```

Comment:
same question as above for core, why a different version is needed for common

Reply:
just for internal progress.